data/product.d/rhel.conf

```diff
@@ -3,6 +3,19 @@
 [Product]
 product_name = Red Hat Enterprise Linux
 
+[Anaconda]
```

Comment:
What it does it to make sure the Subscription module is run once it's upstream code lands in RHEL in the future. 

The base config file has the same list of modules already, minus the Subscription module & as we can't append to values when when override them in the config, we need to set the whole list here even if we want to add a single value.

But good point, I think we should make this at least a separate commit with the explanation above attached so that it is evident why the set of modules to be run for RHEL has been configured in such a way.

Reply:
Moved to a separate commit.